### ISSUE TITLE
make sure that privately reported issues do not show up in the public image folder

### ISSUE THREAD

**Issue by DonnieBLT** [OWASP/BLT#1685]

**Comment by HanilJain** [comment]
it would be really helpful if you could provide a link.

**Comment by HanilJain** [comment]
fixed #1691

**Comment by DonnieBLT** [comment]
I think the best way to do this would be to create a new private bucket

**Comment by HanilJain** [comment]
what does exactly new private bucket means ?

**Comment by DonnieBLT** [comment]
A private bucket, in the context of cloud computing and storage, typically refers to a storage container within a cloud storage service that is designed to hold data. Unlike public buckets, which can be accessed by anyone with the right URL, private buckets are restricted and can only be accessed by specific, authorized users or systems. 

Key features of a private bucket include:

1. **Access Control:** The owner of the bucket can set permissions to control who can view, upload, or download data from the bucket.

2. **Security:** Private buckets often have enhanced security measures like encryption, both at rest and in transit, to protect sensitive data.

3. **Data Integrity:** They often include features to ensure the integrity of the data stored, like versioning and checksums.

4. **Integration:** These buckets can be integrated with other cloud services for data processing, analysis, or backup.

5. **Cost:** The cost of using a private bucket can vary based on the amount of data stored, the level of access control, and additional security features.

Private buckets are commonly used by businesses and individuals to store sensitive data like personal information, confidential business documents, or proprietary data, ensuring that it's not publicly accessible or vulnerable to unauthorized access.

**Comment by HanilJain** [comment]
@DonnieBLT I think this issue can be closed, since #1691 is merged

**Comment by DonnieBLT** [comment]
No, this issue is still valid - we'll have a separate private bucket for private issues
